### ISSUE TITLE
Split current Clients view into Clients and Sessions

### ISSUE THREAD

**Issue by jfmartinezm** [Libki/libki-server#303]
The current Clients view shows, for each client, a lot of related information that belongs to the user or the current session rather than to the client itself. That leads to a very long row (that may not even fit on screens with smaller sizes and forces the user to scroll) and a lot of action buttons under it when selected. 

Maybe it could be simplified by splitting that view and creating two tabs, one for the Clients themselves and other for the current Sessions. The session list could have links on the username and client fields that would lead to the client or user details when clicked.

**Comment by kylemhall** [comment]
Good idea. This would be a good opportunity to use the latest DataTables API and look at updating our existing DataTables. Currently the JavaScript relies heavily on the order in which the columns appear, which is just bad practice ( especially since we hide certain columns based on system settings ).

**Comment by sekjal** [comment]
Have to defer this one to a later release
